docs/docsite/rst/userguide/projects.rst

```diff
@@ -472,14 +505,17 @@ You can create different repos with different namespaces/collections in them. Bu
 6. If the |ah| has self-signed certificates, click the toggle to enable the setting **Ignore Ansible Galaxy SSL Certificate Verification**. For **public Automation Hub**, which uses a signed certificate, click the toggle to disable it instead. Note this is a global setting:
 
 .. image:: ../common/images/settings-jobs-ignore-galaxy-certs.png
+   :alt: setting of jobs ignore galaxy certs
 
 7. Create a project, where the source repository specifies the necessary collections in a requirements file located in the ``collections/requirements.yml`` file. Refer to the syntax described in the corresponding `Ansible documentation <https://docs.ansible.com/ansible/latest/user_guide/collections_using.html#install-multiple-collections-with-a-requirements-file>`_.
 
 .. image:: ../common/images/projects-add-ah-source-repo.png
+   :alt: add ah source repo
```

Comment:
May I suggest a more descriptive text here, like:
"The URL for the Source Control URL in the Type Details section of the Create New Project form."